src/index.js

```diff
@@ -412,7 +412,7 @@ export function check(pretty = true) {
 		}
 
 		// Check if default is set
-		if (opts.default) {
+		if (opts.default != null) {
```

Comment:
Ah, good catch. What if we just removed the `default` key from the `defaultOpts` object? I'm  just concerned that if someone wants the default to be `null` we'll just be introducing the same problem with a different value…